bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -548,7 +548,11 @@ impl<'a> CircuitInputStateRef<'a> {
         if !found {
             return Err(Error::AccountNotFound(sender));
         }
-        Ok(get_contract_address(sender, account.nonce))
+
+        // TODO: figure out why this hack is needed!! maybe the nonce isn't initialized
+        // correctly in tests/traces/bridge/05.json?
+        let nonce = account.nonce + (account.code_hash == H256(*EMPTY_HASH)).to_word();
```

Comment:
I'm not sure why this adjustment is needed, but in the trace at tests/traces/bridge/05.json, we call CREATE from an account that the bus mapping thinks has nonce = 0, but the resulting address that gets pushed to the stack was calculated with nonce = 1.